circuits/cpp/barretenberg/cpp/src/barretenberg/honk/sumcheck/polynomials/barycentric_data.hpp

```diff
@@ -3,6 +3,12 @@
 #include <algorithm>
 #include <array>
 
+// TODO(#674): We need the functionality of BarycentricData for both field and field_t. The former is "literal" i.e. is
+// compatible with constexpr operations, and the former is not. The functions for computing the pre-computable arrays in
```

Comment:
hm literal is a confusing term - i think native (which, at least for me, means non-circuit stuff) is more appropriate

Reply:
ah! the more you know about c++. Are concepts all together being deprecated?